app/models/generic_file.rb

```diff
@@ -137,6 +142,24 @@ def export_as_chicago_citation
     add_doi_to_citation(citation)
   end
 
+  def unexportable?
+    collection_ids = collections.map(&:id)
+
+    # if record is publc, has a doi, and is either in the gv black photograph sub collection or is not in the greater gv black collection
+    public? && !doi.empty? && (collection_ids.include?(GV_BLACK_PHOTOGRAPH_SUB_COLLECTION_ID) || !collection_ids.include?(GV_BLACK_COLLECTION_ID))
+  end
```

Comment:
:warning: By this rule : A public Journal of Pediatrics paper with a DOI is not exported

`public?` true  &&
`!doi.empty?` true  &&
(
  `collection_ids.include?(GV_BLACK_PHOTOGRAPH_SUB_COLLECTION_ID)` false  ||
  !collection_ids.include?(GV_BLACK_COLLECTION_ID) true
)

So we get: true && true && true -> true for `unexportable?`which should not be the case.

Also I think we want to use the definition of "public" as being open access. For inveniordm that is currently what is being marked as [public in the export](https://github.com/galterlibrary/digital-repository/blob/5004f4dcb4ef24de9b346b641df6e910bf35d4f3/app/models/invenio_rdm_record_converter.rb#L171). @sharpattack can confirm if this is correct or not. 

**The criterion**
My understanding is that we export everything, except for 
- any open access record that doesn't have a DOI
- any record in the "old gv black collection" (see https://github.com/galterlibrary/digital-repository/issues/965). This one in particular has to be identified correctly. The metadata meeting notes + collections spreadsheet seem to be good starting points: https://northwestern.app.box.com/file/883830663475?s=lw9txtw8v2rtp4np37apg88j5qypyq9n

Reply:
Ah, nice catch. I thought there were going to be some problems with the logic TBH. It's kind of an awkward set of rules.